src/SDKs/SqlManagement/Management.Sql/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -7,12 +7,13 @@
 		<PackageId>Microsoft.Azure.Management.Sql</PackageId>
 		<Description>Azure SQL Management SDK library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Sql</AssemblyName>
-		<Version>1.20.0-preview</Version>
+		<Version>1.21.0-preview</Version>
```

Comment:
revert, latest published version is still 1.19.0. see https://www.nuget.org/packages/Microsoft.Azure.Management.Sql